prime-router/src/main/kotlin/Sender.kt

```diff
@@ -18,7 +19,9 @@ open class Sender(
     val schemaName: String,
     val keys: List<JwkSet>? = null, // used to track server-to-server auths for this Sender via public keys sets
     val processingType: ProcessingType = ProcessingType.sync,
-    val allowDuplicates: Boolean = true
+    val allowDuplicates: Boolean = true,
```

Comment:
Update the kdoc with the new attributes.  You may need to add them all.

Reply:
done